examples/custom-with-collector-ts/src/index.ts

```diff
@@ -0,0 +1,63 @@
+import { context, propagation, trace } from '@opentelemetry/api';
+import { HoneycombWebSDK } from '@honeycombio/opentelemetry-web';
+import { getWebAutoInstrumentations } from '@opentelemetry/auto-instrumentations-web';
+
+const tracing = () => {
+  const sdk = new HoneycombWebSDK({
+    // To send direct to Honeycomb, set API Key and comment out endpoint
+    // apiKey: 'api-key',
+    endpoint: 'http://localhost:4318', // send to local collector
+    serviceName: 'web-distro',
+    debug: true,
+    skipOptionsValidation: true,
+    resourceAttributes: { 'app.environment': 'development' },
+    instrumentations: [getWebAutoInstrumentations()], // add auto-instrumentation
+  });
+  sdk.start();
+};
+
+const createButton = (text: string, onClick: { (): void; (): void }) => {
```

Comment:
I'm wondering if we can just have this button in the HTML directly instead of creating it here since that will be a more real world use case? I think this was leftover weirdness from when I made the example-greeting-service web example and I didn't have a good HTML file 🙈

Reply:
ha that's a very valid point, updated